packages/ckeditor5-table/src/tablecolumnresize/tablecolumnresizeediting.js

```diff
@@ -346,29 +287,75 @@ export default class TableColumnResizeEditing extends Plugin {
 	}
 
 	/**
-	 * Registers the mouse event listeners for `mousedown`, `mousemove` and `mouseup` events.
+	 * Registers table column resize converters.
 	 *
 	 * @private
 	 */
-	_setupResizingListeners() {
+	_registerConverters() {
+		const editor = this.editor;
+		const conversion = editor.conversion;
+		const widthStyleToTableWidthDefinition = {
+			view: {
+				name: 'figure',
+				key: 'style',
+				value: {
+					width: /[\s\S]+/
+				}
+			},
+			model: {
+				name: 'table',
+				key: 'tableWidth',
+				value: viewElement => viewElement.getStyle( 'width' )
+			}
+		};
+		const tableWidthToWidthStyleDefinition = {
+			model: {
+				name: 'table',
+				key: 'tableWidth'
+			},
+			view: width => ( {
+				name: 'figure',
+				key: 'style',
+				value: {
+					width
+				}
+			} )
+		};
+
+		conversion.for( 'upcast' ).attributeToAttribute( widthStyleToTableWidthDefinition );
+		conversion.for( 'upcast' ).add( upcastColgroupElement( editor.plugins.get( 'TableUtils' ) ) );
+
+		conversion.for( 'downcast' ).attributeToAttribute( tableWidthToWidthStyleDefinition );
+		conversion.for( 'downcast' ).add( downcastTableColumnWidthsAttribute() );
+	}
+
+	/**
+	 * Registers the listeners for the `mousedown`, `mousemove` and `mouseup` events.
+	 *
+	 * @private
+	 */
+	_registerResizingListeners() {
 		const editingView = this.editor.editing.view;
 
 		editingView.addObserver( MouseEventsObserver );
 		editingView.document.on( 'mousedown', this._onMouseDownHandler.bind( this ), { priority: 'high' } );
 
-    this._domEmitter.listenTo( global.window.document, 'mousemove', throttle( this._onMouseMoveHandler.bind( this ), 50 ) );
+		this._domEmitter.listenTo( global.window.document, 'mousemove', throttle( this._onMouseMoveHandler.bind( this ), 50 ) );
 		this._domEmitter.listenTo( global.window.document, 'mouseup', this._onMouseUpHandler.bind( this ) );
 	}
 
 	/**
-	 * Handles the `mousedown` event on column resizer element.
+	 * Handles the `mousedown` event on column resizer element:
+	 *  * calculates the initial column pixel widths;
```

Comment:
We're pretty inconsistent in terms of punctuation for lists in our API docs. However, we typically go either with comma or a dot.

Let's use comma to be more consistent.

```suggestion
	 *  * calculates the initial column pixel widths,
```